user_guide_src/source/outgoing/response/018.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-$date = DateTime::createFromFormat('j-M-Y', '15-Feb-2016');
+$date = DateTime::createFromFormat('!j-M-Y', '15-Feb-2016');
 $response->setDate($date);
```

Comment:
I do not know in which case such a sample code would be used.
https://codeigniter4.github.io/CodeIgniter4/outgoing/response.html#CodeIgniter\HTTP\Response::setDate
So I also don't know the change is correct or not.